data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1478 @@
+			`	"Captain!" Jeff shouts, "An answer to our prayers. Would you be interested in joining our convoy? We're moving supplies to that new Navy base again. 52 tons of cargo and one bunk is all we need of you, and you'll join our fleet."`
+			`	The moment he mentions where the supplies are being moved, the Navy officers glare at him and shake their heads.`
+			`	"If you're trying to get us ambushed, you're doing a good job," one of them says. "Maybe don't tell everyone where we're going?"`
+			` 	The captain of the Alba Oro and Jeff explain together that you have worked with them before.`
+			choice
+				`	"Sure!"`
+				`	"Sorry, but I go my own way."`
+					decline
+			`	They thank you for your assistance, and you give them a hand pushing the heavy cart into your cargo bay.`
+				accept
+
+	on accept
+		log "Decided to help Jeff with transporting cargo and an observer to <destination>."
+
+	on complete
+		"assisted republic" ++
+		payment
+		payment 120000
+		log "The Navy officer from earlier said they'd put in a good word for me."
+		dialog `As you land, Jeff pays you <payment>, and the officer you shuttled lets you know they will put in a good word for you.`
+
+mission "Joining the Navy"
+	autosave
+	name "Navy Candidate"
+	description "Destroy the pirate vessel named <npc> near the <system> system and return to <planet> for an opportunity to join the Republic Navy."
+	source
+		government "Republic"
+	destination "Geminus"
+	on fail
+		dialog `You have failed an essential Navy mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	to offer
+		has "event: war begins"
+		has "event: enlistment begins"
+		not "event: naval employment earned"
+		has "enlistment announced: offered"
+		not "chosen sides"
+	on offer
+		conversation
+			`Your ship's communicator lights with an official announcement from the Republic:`
+			``
+			`	"Hello, Captain <last>," the message begins, "as of now, the Republic Navy is seeking captains skilled in combat to serve in the Navy. However, to discourage people hoping for handouts, applicants need to prove that they want to serve in the Navy. Recently, we've been having issues with a pirate vessel called the <npc> - our intel reports that it was spotted skulking around the <system> system. Destroy the <npc>, and return to <destination> to be formally accepted into the Navy."`
+			choice
+				`	I'll take on that ship.`
+					goto jobAccept
+				`	Sorry, but I have no desire to join the Navy at this time.`
+					defer
+                    
+			label jobAccept
+			`	"We look forward to seeing you on <destination> once the mission is over," the message reads. "Good luck!"`
+					accept
+
+	on accept
+		log "Accepted an enlistment offer from the Republic Navy."
+            
+	npc kill
+		personality heroic staying nemesis target
+		government "Pirate"
```

Comment:
```suggestion
		government "Bounty"
```
This ship just gets obliterated out of the sky with all the Navy capital ships around!